reasoner/processor/AbstractProcessor.java

```diff
@@ -182,6 +185,36 @@ public Driver<Monitor> monitor() {
             return monitor;
         }
 
+        public ReasonerPerfCounters perfCounters() {
+            return perfCounters;
+        }
+
+        public static class ReasonerPerfCounters {
+
+            public static final String KEY_TIME_PLANNING_MS = "time_planning_ms";
+            public static final String KEY_COUNT_MATERIALISATIONS = "count_materialisations";
+            public static final String KEY_COUNT_CONJUNCTION_PROCESSORS = "count_conjunction_processors";
+            public static final String KEY_COUNT_COMPOUND_STREAMS = "count_compound_streams";
```

Comment:
you don't need to prefix this with `KEY_`, remove.
Also, you should suffix `_COUNT` instead of having them be a prefix (see how you did `_MS`) above.

Reply:
right but we can still put the unit at the end - PROCESSOR_CONJUNCTION_COUNT and PLANNER_PLANNING_TIME_MS

Reply:
Refactored and omitted the count because they're 'counters' anyway.